torch/utils/optimizer.py

```diff
@@ -0,0 +1,26 @@
+"""
+This module contains utility method for mobile model optimization and lint.
+"""
```

Comment:
I would still like to evangelize for this function to be placed somewhere more accessible, ideally in the torch.jit Python module itself, given its impact on the overall performance on mobile.  The difference between running vs neglecting to run this function on a model (potentially because the user is not aware of it) at this point is a factor of at least 2x on FP32.  Otherwise we will be mostly reliant on documentation and samples to publicize the presence of this function.  In my opinion this function is more than a utility - more like a necessity.

Reply:
I am not convinced that by placing in torch.jit namespace it will require less evangelization. We will still have to advertise in best practices for mobile etc. And of course we can move this if it is seems more expedient.
I agree on the part that this should be more of a necessity than utility though.

Reply:
how about we create a new mobile folder like one of other proposal @dreiss had. Having the path like torch.mobile.optimize_for_mobile instead of utility if you both feel utility does not give much of necessity.